django_freeradius/base/models.py

```diff
@@ -260,6 +260,14 @@ class Meta:
     def __str__(self):
         return self.unique_id
 
+    def acct_status_type(self):
```

Comment:
you can now remove this method (after adding `acct_status_type = serializers.CharField(write_only=True)`) to the serializer